src/src/pages/capabilities/summary/index.js

```diff
@@ -54,6 +62,16 @@ function JoinDialog({ name, isSubmitting, onCloseRequested, onSubmitClicked }) {
             been approved by existing members.
           </i>
         </Text>
+        {canBypassMembershipApplications && (
+          <Button
+            variation="danger"
+            style={{ position: "absolute", bottom: "1rem" }}
+            disabled={isSubmitting}
+            onClick={onBypassClicked}
+          >
+            FORCE JOIN (CE)
```

Comment:
think this is a bit overkill to call it FORCE JOIN: endpoint is called /join and is valid for you

Reply:
This was my original suggestion:
In most command line interfaces the '--force' flag means 'ignore usual defenses and do it anyway'.
That is exactly what we are doing here.

The word force also emphasizes that it is not the normal approach and sets it apart from the normal join button that we _just_ pressed.

On the other hand, I also argued for the join button automatically approving memberships for CEs, so I might be completely out of my mind 🤷‍♂️

Reply:
If I recall well, Mikkel and Audun asked for a separate join button for CEs
I have no opinion on this.